pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/pubsub/MercuryOrderPublisher.java

```diff
@@ -29,18 +29,18 @@
 
 @Slf4j
 public class MercuryOrderPublisher {
-    private static MercuryOrderDao mercuryOrderDao;
-    private static ParticipantDao participantDao;
+    private MercuryOrderDao mercuryOrderDao;
+    private ParticipantDao participantDao;
 
```

Comment:
@bskinner I removed the statics you mentioned, Thanks for catching that!

Reply:
Thank you!